patterns/2-structured/crossteam-project-valuation.md

```diff
@@ -66,12 +66,12 @@ Once you have your worst-case bounds you can value your cross-team project outpu
 ## Commentary
 
 Despite the trappings of rigor, this process does not yield an exact way to measure cross-team project output.
-In-practice, though, it does give a framework by which you can make a sound decision at how to fund this work.
+In-practice, however, it does give a framework by which you can make a sound decision at how to fund this work.
 After having good, reasonable data according to the above explanation, you should fund dedicated development hours toward running the project up to _**at least**_ of the lesser of the following three levels:
 
 1.  The raw hours saved by the formula above.  Since we're all sure that the formula will produce a number that is below the true number of hours saved, you can have confidence that funding the project up to that point is a sure win for you.
-1.  The amount of time that it takes to support inner sourced contributions to cross-team projects.  Since the contributor would likely have done the work anyway in a one-off fashion, it is worth it to fund the time it takes to faciliate their work going into a shared location.
-1.  Whatever feels good to you.  One intentional side-effect of having a valuation formula is that it  naturally forces measurement of the key points of usage that provide value to consumers.
+1.  The amount of time that it takes to support inner sourced contributions to cross-team projects.  Since the contributor would likely have done the work anyway in a one-off fashion, it is worth it to fund the time it takes to facilitate their work going into a shared location.
```

Comment:
```suggestion
1.  The amount of time that it takes to support InnerSource contributions to cross-team projects.  Since the contributor would likely have done the work anyway in a one-off fashion, it is worth it to fund the time it takes to facilitate their work going into a shared location.
```

While we are at it, we might as well fix this to the preferred capitalization of InnerSource.